commit 80e782da4448bf14205bcf07a991048e3b859bdb (refs/changes/60/60/2)
Author: Julien Gilli <julien.gilli@joyent.com>
Date:   2016-07-12T17:14:27+00:00 (3 years, 3 months ago)
    
    Test gerrit edit commit message
    
    Line from "edit" from Gerrit's web UI

diff --git a/foo b/foo
new file mode 100644
index 0000000..e69de29
